modules/rosa-create-dedicated-cluster-admins.adoc

```diff
@@ -41,53 +40,3 @@ dedicated-admins   rh-rosa-test-user
 ====
 A Forbidden error displays if user without `dedicated-admin` privileges runs this command.
```

Comment:
While you're in here:
s/A Forbidden error displays if user/A `Forbidden` error displays if a user/

Reply:
Done.